src/main/java/com/pretz/dyntreeserver/service/exceptions/DynTreeValidationException.java

```diff
@@ -0,0 +1,10 @@
+package com.pretz.dyntreeserver.service.exceptions;
+
+import com.pretz.dyntreeserver.service.dto.DynTreeDTO;
+import org.springframework.validation.BindingResult;
+
+public class DynTreeValidationException extends RuntimeException {
+    public DynTreeValidationException(DynTreeDTO dynTreeDTO, BindingResult bindingResult) {
+        super("Fuckup!");
```

Comment:
hmmm 🤔

Reply:
I didn't connect this custom exception to validation logic yet xD super fuckup not possible